Css font feature values api (#42835)
mdn/content: files/en-us/web/api/cssfontfeaturevaluesmap/size/index.md

---
title: "CSSFontFeatureValuesMap: size property"
short-title: size
slug: Web/API/CSSFontFeatureValuesMap/size
page-type: web-api-instance-property
browser-compat: api.CSSFontFeatureValuesMap.size
---

{{APIRef("CSSOM")}}

> The **`size`** read-only property of the {{domxref("CSSFontFeatureValuesMap")}} interface returns an unsigned long integer containing the size of the `CSSFontFeatureValuesMap` object.

Reviewer: For all cases - we don't mention data types that are irrelvant to JavaScript. This should be

```suggestion
The **`size`** read-only property of the {{domxref("CSSFontFeatureValuesMap")}} interface returns a positive integer containing the size of the `CSSFontFeatureValuesMap` object.
```

## Value

> An unsigned long integer.

Reviewer: ```suggestion
A positive integer.
```
